docs/VnVPlan/VnVPlan.tex

```diff
@@ -310,6 +310,83 @@ \subsubsection{Area of Testing2}
 
 ...
 
+\subsubsection{Security Requirements}
+
+\begin{enumerate}
+\item{NFRT-SR1\\}
+
+Type: Functional, Manual, Static
+
+Initial State: User is not logged in or terms are not accepted.
+                
+Input: User tries to access the application's database.
+                
+Output: Access will be denied until the user accepts terms and services.
+                
+How test will be performed: Attempts will take place to access database without accepting terms. The team will confirm denial of access. Then, attempt access after accepting terms and services. The team will confirm successful database access.
```

Comment:
...to access **the** database...

Also rather than "the team", can you say "the tester" or "Tester" for consistency with everyone else?

Reply:
resolved